specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/schema/mqtt.kafka.schema.patch.json

```diff
@@ -143,13 +143,15 @@
                                                                 "topic":
                                                                 {
                                                                     "title": "Topic",
-                                                                    "type": "string"
+                                                                    "type": "string",
+                                                                    "pattern": "^(\\/?([+\\w{}-]+)(\\/[+\\w{}-]+)*(\\/#)?|#|\\/|\\$SYS(\\/[+\\w{}-]+)*(\\/#)?)$"
```

Comment:
This matches `abc/def/++/jkl` which is invalid.
The `+` should be the only thing between `/+/` or else word characters.

Also, this pattern does _not_ match `abc/def//jkl` which should be valid by spec.